<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>35</width>
    <height>35</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目标 </string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Import"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>绘制</string>
    </property>
    <addaction name="action_Wireframe"/>
    <addaction name="action_Face"/>
    <addaction name="action_WireframeAndFace"/>
    <addaction name="action_BoundBox"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>显示</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Import"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Wireframe"/>
   <addaction name="action_WireframeAndFace"/>
   <addaction name="action_Face"/>
   <addaction name="action_BoundBox"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>部件列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>部件编辑器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>毁伤树编辑器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>毁伤树绘制器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//open.PNG</normaloff>:/MainWindow/Resources/images//open.PNG</iconset>
   </property>
   <property name="text">
    <string>打开 </string>
   </property>
  </action>
  <action name="action_Wireframe">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//wireframe.PNG</normaloff>:/MainWindow/Resources/images//wireframe.PNG</iconset>
   </property>
   <property name="text">
    <string>线框</string>
   </property>
  </action>
  <action name="action_Face">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//face.PNG</normaloff>:/MainWindow/Resources/images//face.PNG</iconset>
   </property>
   <property name="text">
    <string>面</string>
   </property>
  </action>
  <action name="action_BoundBox">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//boundBox.PNG</normaloff>:/MainWindow/Resources/images//boundBox.PNG</iconset>
   </property>
   <property name="text">
    <string>包围盒</string>
   </property>
  </action>
  <action name="action_WireframeAndFace">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//flatLines.PNG</normaloff>:/MainWindow/Resources/images//flatLines.PNG</iconset>
   </property>
   <property name="text">
    <string>线框和面</string>
   </property>
  </action>
  <action name="action_PartTree">
   <property name="text">
    <string>部件树</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//save.PNG</normaloff>:/MainWindow/Resources/images//save.PNG</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//exit.PNG</normaloff>:/MainWindow/Resources/images//exit.PNG</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_PartEditor">
   <property name="text">
    <string>部件编辑器</string>
   </property>
  </action>
  <action name="action_Import">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//import.PNG</normaloff>:/MainWindow/Resources/images//import.PNG</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_Key">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//key.PNG</normaloff>:/MainWindow/Resources/images//key.PNG</iconset>
   </property>
   <property name="text">
    <string>设为关键部件</string>
   </property>
   <property name="toolTip">
    <string>设为关键部件</string>
   </property>
  </action>
  <action name="action_LessKey">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/images//lessKey.PNG</normaloff>:/MainWindow/Resources/images//lessKey.PNG</iconset>
   </property>
   <property name="text">
    <string>设为非关键部件</string>
   </property>
   <property name="toolTip">
    <string>设为非关键部件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
